pytorch_lightning/utilities/imports.py

```diff
@@ -52,7 +52,7 @@ def _compare_version(package: str, op, version) -> bool:
 _TORCH_LOWER_EQUAL_1_4 = _compare_version("torch", operator.le, "1.5.0")
 _TORCH_GREATER_EQUAL_1_6 = _compare_version("torch", operator.ge, "1.6.0")
 _TORCH_GREATER_EQUAL_1_7 = _compare_version("torch", operator.ge, "1.7.0")
-_TORCH_QUANTIZE_AVAILABLE = _module_available('torch.ops.quantized')
+_TORCH_QUANTIZE_AVAILABLE = hasattr(torch.ops, "quantized")
```

Comment:
Torch 1.4.0 already has quantization. Should we drop this check instead?

Reply:
I am checking what is the issue and turned out that mac for some reason has 
```
torch.backends.quantized.supported_engines <- ['none']
```

Reply:
that's weird 🤯